doc/data/messages/m/missing-format-argument-key/related.rst

```diff
@@ -0,0 +1,2 @@
+`PEP 3101 <https://peps.python.org/pep-3101/>`_
```

Comment:
@Pierre-Sassoulas  @gunungpw I didn't check this myself, but the `related.rst` documents should preferably be a list using `- ...`. That gets formatted a little nicer.

Reply:
There's an issue with lists in something we merged recently see additional detail : https://pylint.pycqa.org/en/latest/messages/warning/arguments-differ.html. I think because it's rst formatting we need ``*`` instead of ``-``.